src/gui/mainwindow.ui

```diff
@@ -248,33 +248,45 @@
     <string>Set Global Upload Limit...</string>
    </property>
   </action>
-  <action name="actionBottomPriority">
+  <action name="actionBottomQueuePos">
    <property name="text">
-    <string>Minimum Priority</string>
+    <string>Bottom of Queue</string>
+   </property>
+   <property name="toolTip">
+    <string>Move to the bottom of the queue</string>
    </property>
    <property name="visible">
     <bool>true</bool>
    </property>
   </action>
-  <action name="actionTopPriority">
+  <action name="actionTopQueuePos">
    <property name="text">
-    <string>Top Priority</string>
+    <string>Top of Queue</string>
+   </property>
+   <property name="toolTip">
+    <string>Move to the top of the queue</string>
    </property>
    <property name="visible">
     <bool>true</bool>
    </property>
   </action>
-  <action name="actionDecreasePriority">
+  <action name="actionDecreaseQueuePos">
    <property name="text">
-    <string>Decrease Priority</string>
+    <string>Lower in Queue</string>
```

Comment:
@thalieht:  Hmm.  In retrospect, I think `Drop in Queue` would read a little better, in this case.

Reply:
Doesn't sound better to me but that's because for some reason i expect "from" after "Drop" when reading it fast. Sounds stupid, i know, and if you're sure i'll change it.

Reply:
@thalieht:  I just thought "lower" seems a mouthful to say.  "Drop" means the same thing, but uses only one syllable instead of two—and four letters instead of five. 

If you truly think the shortened form will be be confusing (despite the tooltip), feel free to keep the label as `Lower`.  :}

Reply:
@thalieht:  I just thought "lower" seems a mouthful to say.  "Drop" means the same thing, but uses only one syllable instead of two—and four letters instead of five. 

If you truly think the shortened form will be be confusing (despite the tooltip), feel free to keep the label as `Lower`.  :}

Reply:
I don't have a strong opinion about this and unfortunately i don't know what other people will think of it.

Reply:
@Chocobo1 @glassez @Piccirello any opinions on this?

Reply:
I'm fine with both. LordNyriox version does sound terser than current one.

Reply:
Sorry, I have no strong opinion since I'm not native English speaker. I don't understand such nuances.

Reply:
Would everyone hate it if I said “Decrease in queue” sounded better? :)

I don’t think drop is the right word- it sounds more like a pop operation.

Reply:
Personally i don't like "decrease". Maybe just copy utorrent and be done with it? 
"Move Up/Down Queue"

Reply:
@thalieht:  `Move Up Queue` / `Move Down Queue` does sound nice.  You can go ahead and use that.  :}

Reply:
Changed.